platform/source/SysTimer.cpp

```diff
@@ -29,6 +29,10 @@ extern "C" {
 #endif
 }
 
+using namespace std::chrono;
+
+const milliseconds deep_sleep_latency{MBED_CONF_TARGET_DEEP_SLEEP_LATENCY};
```

Comment:
For my information: Any particular reason that's not constexpr?

Reply:
No, no particular reason. It should be.

You don't *have* to put `constexpr` instead of `const` on *variables*, as const-qualified variables with constant initialisers are permitted in constant expressions, and hence usable in `constexpr` context.

This retains compatibility with the traditional C++ extension over C that `const int x = 4` can be used in constant expressions.

This is different to function calls, which can never be used in constant expressions if they're not marked `constexpr`.

But it's good to stick the `constexpr` on the variable as it forces an immediate check that `deep_sleep_latency` is usable in a constant expression - you'd get a compile error if `milliseconds`' constructor wasn't constexpr.